Daytime-Operations/Auxiliary-Telescope/AT-Daytime-Checkout/Daytime Checkout SAL Scripts.rst

```diff
@@ -0,0 +1,93 @@
+.. |author| replace:: *E. Dennihy*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _AT-Daytime-Checkout-Daytime-Checkout:
+
+############################
+Daytime Checkout SAL Scripts
+############################
+
+To ensure the Auxiliary Telesocpe is fully operational and ready for nighttime use, we have created a series of SAL Scripts that successively 
+checkout each of the major components, culminating in a full test of the system involving slewing to a target, tracking, and taking a test image
+with LATISS. The goal of these scripts is to test the functionality of each component within the current software environment,
+and alert the daytime staff to any operational issues before going on-sky. These tests are designed to be run during the 
+daytime with an Observing Specialist or other personnel available to verify their results. 
+
+The scripts are intended to be run in order, but offer some flexibility in what state to leave the Auxiliary Telescope (e.g. 
+ready to take calibrations, ready to observe at night, or simply parked and shutdown). They can also be run individually if a 
+single component is undergoing maintenence and requires a check of basic functionality. 
+
+None of the scripts require configuration. 
+
+.. Important::
+
+    The daytime checkout scripts involve Telescope, Dome, and Instrument component movement. You must announce that you are running
+    the checks on the #summit-announce slack channel before execution. 
+
+The order of script execution is nominally:
+
+* enable_latiss - Can be skipped if LATISS enabled 
+* latiss_checkout
+* enable_atcs - Can be skipped if ATCS already enabled
+* atpnuematics_checkout
+* telescope_and_dome_checkout
+* slew_and_take_image_checkout
+* prepare_for/flats - Can be substituted for other prepare_for scripts to leave AuxTel in different state
+
+*enable_latiss*
+===============
+
+Bring LATISS to an enabled state to prepare for checkout. 
+
+*latiss_checkout*
+=================
+
+This script will take a bias and engineering frame with latiss and ensure that they are ingested by the OODS.
+
+Check that the bias pops up in the monitor (https://roundtable.lsst.codes/rubintv/summit/auxtel/monitor_current)
+If it fails, let Patrick or Merlin know
+
+You can check that it ingested at USDF with this link: ******* <b>Add when available<b>
+
+*enable_atcs*
+=============
+
+Bring the ATCS into an enabled state to prepare for checkout. 
+
+*pnuematics_checkout*
+=====================
+
+This script will raise and lower the mirror, exercising the air pressure system. It will also open and close the mirror covers and vents. 
+
+*telescope_and_dome_checkout*
+=============================
+
+This script will move the telescope and dome. You MUST announce on the #summit-announce slack channel before executing this script
+as it results in movement. 
+
+*slew_and_take_image*
```

Comment:
I think here the full name of the script should be shown slew_and_take_image_checkout.